query/security/ldap/src/test/groovy/org/codice/ddf/admin/ldap/discover/LdapTestClaimMappingsSpec.groovy

```diff
@@ -254,6 +254,69 @@ class LdapTestClaimMappingsSpec extends Specification {
         report.messages()*.path as Set == failedPaths
     }
 
+
+    def 'fail when claims mapping value format is incorrect'() {
+        setup:
+        def failedPaths = [badPaths.badClaimMappingPath + '0' + ClaimsMapEntry.VALUE_FIELD_NAME,
+                           badPaths.badClaimMappingPath + '2' + ClaimsMapEntry.VALUE_FIELD_NAME,
+                           badPaths.badClaimMappingPath + '3' + ClaimsMapEntry.VALUE_FIELD_NAME] as Set
+
+        def claimsMapping = createClaimsMapping(ImmutableMap.of("claim1", "space in between",
```

Comment:
I'd be in favor of adding all these cases to a `LdapAttributeName` test class, and then in this class just use one of these to verify this function calls validate on the LdapAttributeName.